static/index.html

```diff
@@ -95,6 +98,32 @@ <h3 id="title"></h3>
                 document.getElementById('thumbnail').src = json["nowPlaying"]["thumbnail"]
             }
         })
+
+        async function OnUrlChange(e) {
+            const url = e.target.value;
+            const reg = /https:\/\/www\.youtube\.com\/(playlist|watch)\?(list|v)=\w+/;
+            if (
+                reg.test(url) && 
+                !rateLimiterCheckbox.checked
```

Comment:
lets add a new case in here, lets say the user already requested a url, but they enter a new one.

the current logic wont show this new thumbnail. but if they hit the enter key, lets override the rate limit and send a new metadata request